apt-requirements.txt

```diff
@@ -30,6 +30,7 @@ libftdi1-dev
 libncursesw5
 libpcsclite-dev
 libssl-dev
+libtool
```

Comment:
#25517 adds the sc_hsm library to the ZIP package which has a non-hermetic system dependency on libtool. This wasn't exercised in CI previously, so the failure is showing up now in this PR.